contracts/adapters/alchemix/AlchemixAdapter.sol

```diff
@@ -0,0 +1,161 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.7.6;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../../interfaces/IUniswapV2Pair.sol";
+import "../../interfaces/IUniswapV2Router02.sol";
+import "../../libraries/UniswapV2Library.sol";
+import "../../BaseAdapter.sol";
+import "./IStakingPools.sol";
+
+contract LuaAdapter is BaseAdapter {
+    address constant MASTER_VAMPIRE =
+        0x12B7b9e21Ad9D7E8992e0c129ED0bccEaC185c3E;
+    IStakingPools constant ALCX_STAKINGPOOLS =
+        IStakingPools(0xAB8e74017a8Cc7c15FFcCd726603790d26d7DeCa);
+    IERC20 constant ALCX = IERC20(0xdBdb4d16EdA451D0503b854CF79D55697F90c8DF);
+    IUniswapV2Pair constant ALCX_WETH_PAIR =
+        IUniswapV2Pair(0x352E5EeE2D9C957710be656534D51Fbb3Ce074d6);
+
+    constructor(address _weth, address _factory) BaseAdapter(_weth, _factory) {}
```

Comment:
Update with latest changes in other adapters where master vampire address is passed as constructor arg